framework/doc/content/source/auxkernels/TagVectorArrayAux.md

```diff
@@ -0,0 +1,17 @@
+# TagVectorArrayAux
+
+Creates an auxiliary field representing the value of a tagged vector for a given
+array variable (specified through the `v` parameter), and given array component
+(specified through the `component` parameter). The family and order of the
+auxiliary variable that this object populates must match the family and order of
+the array variable specified through `v`. Note that the only allowed execute-on
+option for this object is `timestep_end` since residual evaluation must be
+completed before reliable indexing of the tagged vector can be performed.
```

Comment:
We can relax this if the tagged vector is a solution vector.

Reply:
do we have an api to check whether a tag is a solution or residual tag?